build.sbt

```diff
@@ -1,14 +1,11 @@
 name := "fm-sbt-s3-resolver"
-
 organization := "com.frugalmechanic"
-
 version := "0.12.0-SNAPSHOT"
-
 description := "SBT S3 Resolver Plugin"
-
 licenses := Seq("Apache License, Version 2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
-
 homepage := Some(url("https://github.com/frugalmechanic/sbt-s3-resolver"))
+sbtPlugin := true
+crossSbtVersions := Seq("0.13.16", "1.0.0")
```

Comment:
Does this make sense to bump to `1.0.1`?

Reply:
Yes indeed